backend/.env.betanet

```diff
@@ -0,0 +1,13 @@
+# BetaNet backend env file
+
+# TODO: Update below config for betanet
```

Comment:
isn't it already updated?

Reply:
No, we haven't deployed our app on any network.